doc/gov/chapters/chapter09.rst

```diff
@@ -143,7 +143,7 @@ Second, the participant will build/setup/configure the environment space using t
 Expectation 2: Execute the RC1 or RC2 Test suites
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Anuket will provide the participants with the community :doc:`ref_cert_RC1:index` or :doc:`ref_cert_RC2:index` test suites. The participants will execute test cases per instructions and record the quantitative results.
+Anuket will provide the participants with the community :doc:`ref_cert_RC2:index` test suites. The participants will execute test cases per instructions and record the quantitative results.
```

Comment:
@collivier are you saying to change this:
```suggestion
Anuket will provide the participants with the community :doc:`ref_arch_openstack:chapters/chapter08:conformance` or :doc:`ref_cert_RC2:index` test suites. The participants will execute test cases per instructions and record the quantitative results.
```

Reply:
yes :ref:`ref_arch_openstack:chapters/chapter08:conformance`